src/backend/interfaces/REST/py.server/caliopen_api/user/user.py

```diff
@@ -125,9 +125,6 @@ def collection_post(self):
                                  'curve': key.crv})
         self.request.cache.set(cache_key, session_data)
 
```

Comment:
what happen if `key = None ` ?

Reply:
it crash later as application try to find a curve and it's not OK (device exist but public key was not created when creating the device)

I don't know which error code to raise, instead of only log an error when key is not found. Any idea ?

Reply:
something similar to `raise ValidationError(detail='No device informations')` ?

Reply:
It's not a client error, so use of a 4xx http error is not the best solution and can't find any 5xx error that could fit (or a generic 500 ?).

Reply:
424 ? (event if it is not client error, it is dependency fail, isn't it ?)